sdk/python/tests/facade/test_NemFacade.py

```diff
@@ -71,12 +74,18 @@ def test_cannot_create_around_unknown_network(self):
 	def _create_real_transfer():
 		facade = NemFacade('testnet', AccountDescriptorRepository(YAML_INPUT))
 		transaction = facade.transaction_factory.create({
-			'type': 'transfer',
+			'type': 'transfer_transaction_v1',
 			'signer_public_key': 'TEST',
+			'fee': 0x186a0,
+			'timestamp': 191205516,
 			'deadline': 191291916,
 			'recipient_address': 'TALICE5VF6J5FYMTCB7A3QG6OIRDRUXDWJGFVXNW',
 			'amount': 5100000,
-			'message': 'blah blah'
+			'message_envelope_size': 0x11,
+			'message': {
+				'message_type': 'plain',
+				'message': 'blah blah'.encode('utf8')
+			}
 		})
 		return transaction
 
```

Comment:
would it be possible to add Verifiable and NonVerifiable variants for all these tests?

Reply:
that is related to something I asked ealier: do we want/need create_non_verifiable thing?

Reply:
probably not but should add tests for to_non_verifiable_transaction at least